docs/upload_specifications.ADR.md

```diff
@@ -177,4 +178,9 @@ If any other error occurs, a general 500 error will be reported:
 * Should it be an error to submit multiple files or tabs for the same type? If not,
   how should the diffent files be denoted and ordered?
   * This has follow on effect for how spreadsheet type views in the UI should be displayed.
-* Should we disallow filenames with commas? They may cause problems with the new endpoint.
\ No newline at end of file
+* Should we disallow filenames with commas? They may cause problems with the new endpoint.
+* Should we strictly enforce a column count for every row in xSV files?
+  * Not enforcing a count makes it somewhat easier for users to fill in the data - they don't
+    need to add extraneous commas or tabs to the end of the line.
+  * Enforcing a count makes it less likely that a user will commit silent counting errors if
+    there are many empty entries between items in a line.
```

Comment:
I don't think it is a bad thing to enforce a count - it is mostly dependent on what the row represents and if it needs to be flexible or not. Based on our templates, I think having a limit could be supported with the template and docs, as long as it is appropriate for the use case

Reply:
I added an answer to this question saying that we'll enforce a line count